### PR TITLE
Adds the command multitool

### PR DESCRIPTION
**Pull request by JimKil3**
## What Does This PR Do
Adds the command multitool, a blue variant of the multitool which sits on the bridge all shift, much like its namesake. It has the same stats as a Syndicate multitool (very slightly faster use speed) and a unique suicide act like its inspiration, the medical wrench.

This PR also adds a command multitool to bridge of delta and cerestation, as they didn't have a multitool at all before, and replaces the boring one on box and meta with a shiny blue one.

## Why It's Good For The Game
More flavor and unique things is fun! Also, Delta not having a multitool on the bridge while it has a fireaxe cabinet is a little weird.

## Images of changes
multitool
description

## Testing
Loaded into all the maps, saw the blue multitool.
Used the multitool on the fireaxe cabinet in bridge, then committed suicide with it.

## Changelog
:cl:
add: Added the command multitool
add: Delta and Cerestation now have a command multitool
tweak: Box and Metastation's bridge multitools have been replaced with command ones
/:cl:

### CODE REVIEW

**Comment by YoureNotTheSharpest** [comment]
now where's the science crowbar?

**Review by S34NW — APPROVED** [review]

File: code/game/objects/items/tools/multitool.dm [hunk 1]
```diff
@@ -105,6 +105,33 @@
...
+		return
+
+	user.Stun(10 SECONDS)
+	playsound(loc, 'sound/effects/supermatter.ogg', 50, 1, -1)
+	sleep(20)
+
+	for(var/obj/item/W in user)
+		user.unEquip(W)
+
+	var/obj/item/multitool/command/M = new /obj/item/multitool/command(loc)
```
Comment (S34NW) [inline code/game/objects/items/tools/multitool.dm:128]: Actually, won't this drop the original and make a second one?

Comment (JimKil3) [reply code/game/objects/items/tools/multitool.dm:128]: I'll be honest, I basically copy-pasted this from medical wrench code. I'll take a look in a minute and double-check to make sure this doesn't happen

**Review by S34NW — CHANGES_REQUESTED** [review]
Dupe issue real, please fix (edit the existing multitool, instead of making a new one)

File: code/game/objects/items/tools/multitool.dm [hunk 2]
```diff
@@ -105,6 +105,32 @@
...
+    //basically just cleaned up and copied from the medical wrench code
+	if(!user)
+		return
+
+	user.Stun(10 SECONDS)
+	playsound(loc, 'sound/effects/supermatter.ogg', 50, 1, -1)
+	sleep(20)
+
+	src.add_fingerprint(user)
+	src.desc += " Its screen displays the text \"[user.name]: executed for mutiny.\""
```
Comment (S34NW) [inline code/game/objects/items/tools/multitool.dm:126]:
```suggestion
	add_fingerprint(user)
	desc += " Its screen displays the text \"[user.name]: executed for mutiny.\""
```

**Review by S34NW — CHANGES_REQUESTED** [review]

**Review by S34NW — APPROVED** [review]

File: code/game/objects/items/tools/multitool.dm [hunk 3]
```diff
@@ -105,6 +105,32 @@
...
+	toolspeed = 0.95 //command those wires / that fireaxe cabinet!
+
+/obj/item/multitool/command/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] is attempting to command the command multitool! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+    //basically just cleaned up and copied from the medical wrench code
+	if(!user)
+		return
+
+	user.Stun(10 SECONDS)
+	playsound(loc, 'sound/effects/supermatter.ogg', 50, 1, -1)
```
Comment (Henri215) [inline code/game/objects/items/tools/multitool.dm:122]:
```suggestion
	playsound(loc, 'sound/effects/supermatter.ogg', 50, TRUE, -1)
```

**Review by Henri215 — CHANGES_REQUESTED** [review]
Can you add inhand icon for the command multitool too?

**Pull request merged**